package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "coco-toolbox",
+  "version": "1.0.0",
```

Comment:
This is still a prerelease, so let's set the package version to 0.0.0.

Reply:
You missed this change. In package.json, let's set the `coco-toolbox` package version to 0.0.0.